### PR TITLE
add feature.md entry for "add funds to wallet"

### PR DESCRIPTION
**Pull request by odeac**
## Overview
<sup>_What this PR does, and why it's needed_</sup>
RHOL-1105

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -163,3 +163,22 @@
...
+* send a message  to the wallet process with the structure **("deposit", purse, returnChannel)** where purse is the unforgeable name of a purse containing the rev to deposit in the wallet
+* expect a response on the return channel with a **true** value
+* expect *purse* to be empty (by calling "getBalance")
+* expect the wallet to have the original purse amount (by calling "getBallance")
+
+#### Failed to add funds to a wallet
+##### test: test/test_wallet.py::test_add_funds_failure
+##### steps: 
+* given an existing empty wallet 
+* send a message  to the wallet process with the structure **("deposit", fakePurse, returnChannel)** where fakePurse is an unforgeable name which doesn't implement the purse protocol
```
Comment (KentShikama) [inline docs/features.md]: two spaces between 

message  to

File: docs/features.md [hunk 2]
```diff
@@ -163,3 +163,22 @@
...
 
 * TBD
+
+## Contracts
+### As a user, I want to be able to add REV to my wallet so that I have available REV to pay for goods/services
+#### Successfully add funds to a wallet
+##### test: test/test_wallet.py::test_add_funds_success
+##### steps: 
+* given an existing empty wallet 
+* send a message  to the wallet process with the structure **("deposit", purse, returnChannel)** where purse is the unforgeable name of a purse containing the rev to deposit in the wallet
```
Comment (KentShikama) [inline docs/features.md]: same here

File: docs/features.md [hunk 3]
```diff
@@ -163,3 +163,22 @@
 ##### steps:
 
 * TBD
+
+## Contracts
+### As a user, I want to be able to add REV to my wallet so that I have available REV to pay for goods/services
+#### Successfully add funds to a wallet
+##### test: test/test_wallet.py::test_add_funds_success
+##### steps: 
+* given an existing empty wallet
```
Comment (rabbitonweb) [inline docs/features.md]: new line needed for the `source of truth`

File: docs/features.md [hunk 4]
```diff
@@ -163,3 +163,22 @@
...
+* given an existing empty wallet 
+* send a message  to the wallet process with the structure **("deposit", purse, returnChannel)** where purse is the unforgeable name of a purse containing the rev to deposit in the wallet
+* expect a response on the return channel with a **true** value
+* expect *purse* to be empty (by calling "getBalance")
+* expect the wallet to have the original purse amount (by calling "getBallance")
+
+#### Failed to add funds to a wallet
+##### test: test/test_wallet.py::test_add_funds_failure
+##### steps: 
+* given an existing empty wallet
```
Comment (rabbitonweb) [inline docs/features.md]: newline pls

**Pull request closed without merging**